website/docs/reference/dbt-commands.md

```diff
@@ -4,7 +4,7 @@ title: "dbt Command reference"
 
 dbt is typically run one of two ways:
 * In [dbt Cloud](/docs/get-started/develop-in-the-cloud)
-* On the command line
+* On the [Command Line Interface](/docs/get-started/about-the-cli) (CLI)
```

Comment:
```suggestion
* On the [command line interface](/docs/get-started/about-the-cli) (CLI)
```

Reply:
similar to GUI, which stands for graphical user interface

Reply:
ah ok, i originally had it as lowercased but then i double checked and went with the styling on `[this page](https://docs.getdbt.com/docs/get-started/about-the-cli)` 🙈

would it make sense to open a pr to edit that page above and correct?

Reply:
good flag, @mirnawong1 ! i opened PR #2482 for it